soh/src/overlays/actors/ovl_En_Du/z_en_du.c

```diff
@@ -255,7 +275,13 @@ void func_809FE040(EnDu* this) {
         if (this->unk_1E6 >= 8) {
             this->unk_1E6 = 0;
         }
-        Animation_ChangeByInfo(&this->skelAnime, sAnimationInfo, animationIndices[this->unk_1E6]);
+        // #region SOH[Enhancement]
+        if (CVarGetInteger("gEnhancements.FixDaruniaDanceSpeed", 1)) {
```

Comment:
Considering the rest of the convo, NTSC will likely need a different set of adjustments, because of lag still existing, but yeah, the check could still be there. Honestly, I'm hoping for a better method by the time NTSC support comes around XD